app/views/reminder_mailings/show.html.slim

```diff
@@ -0,0 +1,53 @@
+= render 'section_navigation'
+
+h1 #{@reminder_mailing.t_enum(:kind)} Erinnerungs-Email
```

Comment:
please check the rendering of this file, since the footer is not sticky at the bottom anymore on this view